hal/src/nRF52840/ble_hal.cpp

```diff
@@ -1125,7 +1131,8 @@ int BleObject::Broadcaster::configure(const hal_ble_adv_params_t* params) {
                   bleGapAdvParams.interval*0.625, bleGapAdvParams.duration*10);
         ret = sd_ble_gap_adv_set_configure(&advHandle_, &bleGapAdvData, &bleGapAdvParams);
     }
-    return nrf_system_error(ret);
+    CHECK(nrf_system_error(ret));
```

Comment:
Any reasoning for the change? :)

Reply:
Because it won't log any error if failed to configure advertising set without this `CHECK()`.

Reply:
Also it's fine to log the error where the `configure()` is called.